lib/rubygems/commands/query_command.rb

```diff
@@ -123,6 +123,10 @@ def prerelease?
     options[:prerelease]
   end
 
+  def show_prereleases?
+    options[:prerelease].nil? || options[:prerelease]
```

Comment:
You can reuse the `prerelease?` method here :)

Reply:
how i missed that 😂 thanks!